src/configs/formatters/getTsConfig.ts

```diff
@@ -0,0 +1,58 @@
+import StyleDictionary, { Config, filter, formatHelpers } from 'style-dictionary'
+import { DesignToken } from 'style-dictionary/types/DesignToken'
+
+export function getWebTsConfig(sd: typeof StyleDictionary, brand: string, theme: string): Config {
+  const destination = `index.${theme}.web.ts`
+  const sizeTransform = 'size/pxToRem'
+
+  return getTsConfig(sd, theme, brand, sizeTransform, destination)
+}
+
+export function getMobileTsConfig(
+  sd: typeof StyleDictionary,
+  brand: string,
+  theme: string
+): Config {
+  const destination = `index.${theme}.mobile.ts`
+  const sizeTransform = 'size/px'
+
+  return getTsConfig(sd, theme, brand, sizeTransform, destination)
+}
+
+function getTsConfig(
+  sd: StyleDictionary.Core,
+  theme: string,
+  brand: string,
+  sizeTransform: string,
+  destination: string
+) {
+  sd.registerFormat({
+    name: 'typings/es6',
+    formatter: ({ dictionary, file }) => {
+      const tokens = JSON.stringify(formatHelpers.minifyDictionary(dictionary.tokens), null, 2)
+      return `${formatHelpers.fileHeader({
+        file,
+      })}export const theme = ${tokens} as const;\n`
+    },
+  })
+
+  return {
+    source: [`src/tokens/themes/${theme}.json`, `src/tokens/overrides/${brand}-${theme}.json`],
+    include: ['src/tokens/global/**/*.json'],
```

Comment:
si j'ai bien compris, on prends les tokens `global` puis on les écrasent avec le `theme` puis avec les `overrides`

je trouverais ça plus facile à lire si on les avait dans l'ordre

```suggestion
    include: ['src/tokens/global/**/*.json'],
    source: [`src/tokens/themes/${theme}.json`, `src/tokens/overrides/${brand}-${theme}.json`],
```